packages/cli-plugin-metro/src/commands/start/runServer.ts

```diff
@@ -40,6 +49,39 @@ export type Args = {
 };
 
 async function runServer(_argv: Array<string>, ctx: Config, args: Args) {
+  let port = args.port ?? 8081;
+  const packagerStatus = await isPackagerRunning(port);
+
+  const handleSomethingRunningOnPort = async () => {
```

Comment:
nit: Simplify naming.
```suggestion
  const handlePortUnavailable = async () => {
```